src/component/parser/json/converter/ProcessConverter.ts

```diff
@@ -119,7 +121,7 @@ export default class ProcessConverter extends AbstractConverter<Process> {
     }
 
     if (numberOfEventDefinitions == 1) {
-      const eventDefinition = eventDefinitions[0];
+      const eventDefinition = eventDefinitions.filter(eventDefinition => eventDefinition.counter == 1)[0];
```

Comment:
:question: I don't understand why this is needed
- existing tests are passing
- the map/reduce is supposed to already manage this

Reply:
Because we want to get the eventDefinition where the counter is equal to 1.
The map/reduce just sum all counters in the map to know if there is 0, 1 or more eventDefinitions in the all map.

Reply:
After discussion, the change has been reintroduced after my refactoring; let's keep it for now